glslc/src/main.cc

```diff
@@ -69,10 +69,11 @@ void PrintHelp(std::ostream* out) {
 }
 
 // Parses the argument for the option at index in argv. On success, returns
-// true and writes the parsed argument into option_argument. Returns false
-// if any error occurs. option is the expected option at argv[index].
-// After calling this function, index will be at the last command line
-// argument consumed.
+// true and writes the parsed argument into option_argument. Returns false if
+// any error occurs. The option parameter is the expected option at argv[index].
+// If the option parameter ends with an equal sign, its argument is whatever
+// follows that equal sign, even empty string. After calling this function,
```

Comment:
And if it's not an equal sign, then the argument is either the rest of argv[index] or argv[++index]?